recipes/dcase2022_task4_baseline/README.md

```diff
@@ -32,55 +40,120 @@ The synthetic part of the dataset is composed of synthetically soundscapes, gene
 
-The tensorboard logs can be tested using the command `tensorboard --logdir="path/to/exp_folder"`. 
+The tensorboard logs can be tested using the command `tensorboard --logdir="path/to/exp_folder"`.
+
+
+## **(NEW!)** Energy Consumption
+
+In this year DCASE Task 4 Challenge, we also use energy consumption (kWh)
+via [CodeCarbon](https://github.com/mlco2/codecarbon) as an additional metric to rank the submitted systems.
+
+We encourage the participants to provide, for each submitted system (or at least the best one),
+using [CodeCarbon](https://github.com/mlco2/codecarbon), the following energy consumption figures in kWh:
+
+1) whole system training
+2) devtest inference
+3) evaluation set inference
+
+You can refer to [Codecarbon](https://github.com/mlco2/codecarbon) on how to do this (super simple!)
+or to this baseline code see `local/sed_trained.py` for some hints on 
+how we are doing this for the baseline system.
+
+
+**Important** 
+
+In addition to this, we kindly ask the participants to 
+run also, on the same hardware used for 2) and 3) 
+also energy consumption kWh for:
+
+1) devtest inference for baseline system using: 
+
+`python train_sed.py --test_from_checkpoint /path/to/downloaded.ckpt`
+You can find the energy consumed in kWh in `./exp/2022_baseline/devtest_codecarbon/devtest_tot_kwh.txt`
+
+2) evaluation set inference for baseline system using:
+`python train_sed.py --eval_from_checkpoint /path/to/downloaded.ckpt`
+You can find the energy consumed in kWh in `./exp/2022_baseline/evaluation_codecarbon/eval_tot_kwh.txt`
+
+**Why we require this ?**
+
+Energy consumption depends on hardware and each participant uses
+different hardware. 
+
+To obviate for this difference we use the baseline inference kWh energy consumption 
+as a common reference. Because of this is important that 
+inference energy consumption figures for both submitted system 
+and baseline are computed on same hardware under similar loading. 
+
+
+
+**Common issues:**
+
+If training appears too slow, check with `top` and with `nvidia-smi` that you 
+are effectively using a GPU and not the CPU. 
+If running `python train_sed.py` uses by default the CPU you may have **pytorch** installed 
+without CUDA support. 
+
+Check with IPython by running this pytorch line `torch.rand((1)).cuda()` 
+If you encounter an error install CUDA-enabled pytorch from https://pytorch.org/
+Check again till you can run `torch.rand((1)).cuda()` successfully. 
 
 #### Results:
 
-Dataset | **PSDS-scenario1** | **PSDS-scenario2** | *Intersection-based F1* | *Collar-based F1*
+Dataset | **PSDS-scenario1** | **PSDS-scenario2** | *Intersection-based F1* | *Collar-based F1* 
 --------|--------------------|--------------------|-------------------------|-----------------
-Dev-test| **0.351**          | **0.531**          | 64.1%                   | 41.4%
+Dev-test| **0.352**          | **0.559**          | 78.90%                  | 43.22%
```

Comment:
Maybe we should discuss why we do have different results? 
Are you using the pre-model available on Zenodo in order to have these results or is a new training? Because we have different results showed here and in the website, plus the results currently reported on the GitHub, are the ones coming from the student model, using the pre-trained model available on Zenodo. We should agreed on which results report, so to don't be confused and don't confuse participants.